app/controllers/collections_controller.rb

```diff
@@ -3,11 +3,10 @@
 require 'zip'
 
 class CollectionsController < ApplicationController
-  before_action :load_and_authorize_collection, except: %i[index new create]
+  before_action :load_and_authorize_collection, except: %i[index new create create_ajax]
```

Comment:
Having a separate action for responding to AJAX is a bit uncommon, I'd say. Usually, one would allow an action to respond to both. An alternative would be to route it to a dedicated controller (e.g., scoped controller in an `Ajax` module). The choice depends on the amount of code / logic to be re-used, of course.

How is this handled in other places in your code, so is this consistent?